### PR TITLE
add getReviews

### PR DESCRIPTION
**Pull request by scarlett-k-nhs**

### CODE REVIEW

**Comment by scarlett-k-nhs** [comment]
Just updating this - lines accidentally got deleted when assessing object structure

File: app.test.js [hunk 1]
```diff
@@ -46,4 +46,38 @@ describe('getCategories', () => {
...
+    });
+    test("Responds with review objects with correct properties", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({body}) => {
+          expect(body.reviews.length).toBeGreaterThanOrEqual(0);
+          body.reviews.forEach((review) => {
+              expect(review).toMatchObject({
+                  review_id: expect.any(Number),
```
Comment (mezzDavies) [inline app.test.js:68]: Should should be asserting that all the required properties (listed on the ticket) are present and of the correct data type.

File: app.test.js [hunk 2]
```diff
@@ -46,4 +46,38 @@ describe('getCategories', () => {
...
       });
+})
+
+describe('getReviews', () => {
+  test("Responds with an object containing an array of review objects", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({body}) => {
+          expect(body.reviews).toEqual(expect.any(Array));
```
Comment (mezzDavies) [inline app.test.js:57]: If you have a forEach in your test any assertions that you have an array are implied by the forEach (as it wouldn't work if it wasn't an array). Checking the length of an array is still valid though to assert it contains the expected number of elements.

**Review by mezzDavies — COMMENTED** [review]
Just a little bit of work on the testing please (some assertions can be removed though 😄 ). See my individual comments.

**Pull request closed without merging**